ocpi-toolkit-2.2.1/src/main/kotlin/com/izivia/ocpi/toolkit/modules/credentials/services/CredentialsCommon.kt

```diff
@@ -0,0 +1,52 @@
+package com.izivia.ocpi.toolkit.modules.credentials.services
+
+import com.izivia.ocpi.toolkit.common.OcpiServerNoMatchingEndpointsException
+import com.izivia.ocpi.toolkit.modules.versions.domain.Endpoint
+import com.izivia.ocpi.toolkit.modules.versions.domain.InterfaceRole
+import com.izivia.ocpi.toolkit.modules.versions.domain.ModuleID
+
+class CredentialsCommon {
+    companion object {
```

Comment:
nit: I have a feeling this was code written in Java, and automatically transformed to Kotlin ...  there's barely ever a need for companion objects ... just move the method out of class ... function will be part of package

Reply:
I'm pretty new to Kotlin, so I'll make more mistakes like this 😓

Reply:
I'm by no means an expert ... it looked very much like the code I wrote 3 months ago ;)

That's why I think PR comments are a great way to transmit this info, as it gives real world examples. I hope this didn't discouraged you ... overall great contribution. Also .. I wouldn't call it a mistake (ergo the `nit:`), just room for improvement ;)

Reply:
I'm always open to learn.